src/main/java/com/artipie/npm/proxy/NpmProxyStorage.java

```diff
@@ -23,183 +23,41 @@
  */
 package com.artipie.npm.proxy;
 
-import com.artipie.asto.Concatenation;
-import com.artipie.asto.Content;
-import com.artipie.asto.Key;
-import com.artipie.asto.rx.RxStorage;
 import com.artipie.npm.proxy.model.NpmAsset;
 import com.artipie.npm.proxy.model.NpmPackage;
 import io.reactivex.Completable;
 import io.reactivex.Maybe;
-import io.reactivex.Single;
-import io.vertx.core.json.JsonObject;
-import java.nio.charset.StandardCharsets;
 
 /**
- * NPM Proxy storage.
+ * NPM Proxy storage interface.
  * @since 0.1
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
-public final class NpmProxyStorage {
-    /**
-     * Underlying storage.
-     */
-    private final RxStorage storage;
-
-    /**
-     * Ctor.
-     * @param storage Underlying storage
-     */
-    public NpmProxyStorage(final RxStorage storage) {
-        this.storage = storage;
-    }
-
+public interface NpmProxyStorage {
```

Comment:
@brastak same comment as for `NpmProxyRemoteImpl` to change the name. How about `RxNpmProxyStorage`?